pkg/apis/istio/v1alpha1/common.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Istio Authors
```

Comment:
Suggest moving these out to pkg/apis/istio since they are used on all proto packages.

Reply:
as discussed offline, keep them in both packages, otherwise it would mess up with the import path